eng/Packages.Data.props

```diff
@@ -75,10 +75,11 @@
     <PackageReference Update="Azure.Core" Version="1.9.0" />
     <PackageReference Update="Azure.Core.Amqp" Version="1.0.0" />
     <PackageReference Update="Azure.Core.Experimental" Version="0.1.0-preview.7" />
+    <PackageReference Update="Azure.Messaging.EventHubs" Version="5.3.0" />
```

Comment:
This will be unused until the `Azure.Messaging.EventHubs.Processor` package is released, which will happen after the 5.3.0 release of the core Event Hubs package.